test/WebSites/MvcTagHelpersWebSite/Controllers/MvcTagHelper_HomeController.cs

```diff
@@ -70,5 +72,52 @@ public IActionResult Index()
         {
             return View();
         }
+
+        public IActionResult ProductEdit()
+        {
+            ViewBag.Items = _products;
+
+            return View();
+        }
+
+        public IActionResult Employee()
+        {
+            var employees = new List<Employee>();
+
+            for (int i = 1; i < 5; ++i)
+            {
+                employees.Add(new Employee()
+                {
+                    Name = "EmployeeName_" + i,
+                    Number = i
+                });
+            }
+
+            return View(employees);
+        }
+
+        public IActionResult WarehouseProduct()
+        {
+            ViewBag.Items = new SelectList(_products, "Number", "ProductName", 9);
+
+            return View();
+        }
+
+        public IActionResult WarehouseEmployee()
```

Comment:
though view focuses on `Employee` sub-properties, this is more of an `EditWarehouse()` action